core/themes/basis/theme-settings.php

```diff
@@ -70,3 +70,95 @@
     $form['footer'][$field] = color_get_color_element($form['theme']['#value'], $field, $form);
   }
 }
+
+/**
+ * Altering the theme settings for CSS updates.
+ */
+function basis_form_system_theme_settings_alter(&$form, &$form_state, $form_id = NULL) {
```

Comment:
Do we need to alter this form, can't we change the original? It's Basis's form, isn't it?

Reply:
I've removed this function. Please, review this part of code again to see if there are any organizational issues. 

Also, I'm not quite sure where to put blank lines and such. Where should I look for guidance, because making you give me feedback?

Reply:
> Yes it is Basis's form. I think I was thrown a bit, because I expect the original form to be inside of a function and it wasn't.

Yes, `theme-settings.php` is very weird like that.